tests/e2e/chain/export.go

```diff
@@ -0,0 +1,25 @@
+package chain
```

Comment:
Is there a `doc.go` on this package, `export`? What is it's purpose/intended use-case?

Reply:
We [marshal the data into a json](https://github.com/osmosis-labs/osmosis/pull/1363/files#diff-19b162f07421408062b8543704d0594a559b93da57ff73b40de47def07601274R35-R39) within the docker container in order to [retrieve this data by the e2e test suite](https://github.com/osmosis-labs/osmosis/pull/1363/files#diff-7d53e76218ae6e6128b32ace0c99cf129d5f294a3aac6a6ddb9f115bc85cece2R294-R308). When I initually tried to unmarshal this data however, I ran into an issue where the marshaler did not know how to handle [some sdk structs that dont get exported](https://github.com/osmosis-labs/osmosis/pull/1363/files#diff-f75036128357129bc9f36e9602a2ff277e58e765a5872d493ce96f78ecf03800R38-R41). @p0mvn created this export package in order to export only the required structs needed to initialize the genesis and configs in order to get around this issue.

Reply:
There is a README explaining what this package is for:
https://github.com/osmosis-labs/osmosis/blob/main/tests/e2e/README.md

@czarcas7ic could you check if README needs to be updated after this PR please

Reply:
We can probably add to it by saying that the e2e test runner now does chain initialization in the container by default.

Other updates I noticed:
- there is no `upgrade` package anymore, it was renamed earlier
- state marshaling and unmarshaling on the volume